analyses/fusion-frequencies/01-fusion-frequencies.R

```diff
@@ -0,0 +1,296 @@
+
+
+# Create output dir ------------------------------------------------------------
+
+# Detect the ".git" folder -- this will in the project root directory.
+# Use this as the root directory to ensure proper sourcing of functions no
+# matter where this is called from
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+setwd(root_dir)
+
+# Set path to results, plots, and subset files directories
+data_dir <- file.path(root_dir, "data")
+module_dir <- file.path(root_dir, "analyses", "fusion-frequencies")
+results_dir <- file.path(module_dir, "results")
+
+# Source function
+source(file.path(module_dir,"utils","freq_counts.R"))
```

Comment:
Thanks for pointing that out , I did check for some known genes for example BRAF,MET etc get and `ENSG00000157764` but that must be because of the `filter(!is.na(rmtl), !is.na(version)) ` so only a specific set of genes get annotated by ` ann_ensg_hugo_rmtl_df ` . I'll update that!